codec-http2/src/test/java/io/netty/handler/codec/http2/LastInboundHandler.java

```diff
@@ -99,7 +99,7 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
 
     @Override
     public void channelWritabilityChanged(ChannelHandlerContext ctx) throws Exception {
-        if (writabilityStates == "") {
+        if (writabilityStates.equals("")) {
```

Comment:
if there is possibility of `writabilityStates` to be `null`, it might be better to write this as `"".equals(writabilityStates)`

Reply:
@jplock Thanks! I fixed it.